client/my-sites/site-settings/disconnect-site/style.scss

```diff
@@ -0,0 +1,11 @@
+.disconnect-site__card {
+		width: 70%;
```

Comment:
This should probably use a [breakpoint], it makes sense on wider displays but probably not on something like a phone. Here's an example:

https://github.com/Automattic/wp-calypso/blob/b599ea715444c909bfd8cd1721c0d160e8b1e18e/client/components/dialog/style.scss#L57-L59

Maybe a designer can provide some better feedback 🙂 

[breakpoint]: https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/mixins/_breakpoints.scss